test/predictedspeeds.cc

```diff
@@ -63,7 +52,7 @@ TEST(PredicteSpeeds, test_decoding) {
 
   // Decode the base64 string and cast the data to a raw string of signed bytes
   auto decoded_str = decode64(encoded_speed_string);
-  EXPECT_EQ(decoded_str.size(), 402);
```

Comment:
This original and new value don't make sense to me. There are 200 coefficients each should be 2 bytes wide so we should have 400 here... Are the decoded values different, I noticed the test below doesn't test equality, maybe it was doing this because of the decoding bug you fixed

Reply:
Think I figured this out, noted here: https://github.com/valhalla/valhalla/pull/2452